### ISSUE TITLE
Where are our files

### ISSUE THREAD

**Issue by seltmann** [scan-bugs-org/North-American-Arthropod-Biodiversity#8]
I keep forgetting

![image003](https://user-images.githubusercontent.com/1044474/73880191-cb8f5180-4812-11ea-85ac-94ed84cac798.jpg)

**Comment by neilcobb** [comment]
forgetting?

**Comment by seltmann** [comment]
Yes, there are several locations on Dropbox and Drive. This is just to help point to the most recent versions.

**Comment by neilcobb** [comment]
C:\Users\cobb\Dropbox (Personal)\Biodiversity_North_America_2020\Taxonomy_Databases\Nick_GBIF NA_12_17_2019

Final_Ecology_Codes3 ?
